content/special-topics/color-management/color-spaces.md

```diff
@@ -6,21 +6,27 @@ draft: false
+darktable's input images are either RGB files (like JPEGs or TIFFs) or camera raws. Both store visual information as a combination of primary colours like red, green and blue which, together, describe a light emission to be later recreated by a display.
+
+It is important to mention that colour only exists in the human brain, as a product of the decomposition of a light emission by the cone cells in the retina. What RGB files do is describing light emissions with a similar method, for technical reasons. However, the definition of each primary colour depends on the properties of each RGB space and might look very different to human eyes from what humans call "red", "green" or "blue".
+
+Notice of warning, RGB is not really a color, but rather a light emission being encoded for computational purposes, that will later be physically created by your display, and interpreted as colour only once it enters your brain through your retina. Calling RGB "colour" is a shortcut.
+
+Most of the actual image processing takes place in a large RGB "working profile" space, with some (mostly older) modules internally working in the CIELab 1976 color space (often just called “Lab”). The final output of the image processing pipeline is once again in an RGB space shaped for either monitor display or the output file.
 
 This process implies that the pixelpipe has two fixed color conversion steps: [_input color profile_](../../module-reference/processing-modules/input-color-profile.md) and [_output color profile_](../../module-reference/processing-modules/output-color-profile.md). In addition there is the [_demosaic_](../../module-reference/processing-modules/demosaic.md) step for raw images, where the colors of each pixel are reconstructed by interpolation.
 
 Each module has a position in the pixelpipe which tells you which color space the module lives in:
 
 up to [_demosaic_](../../module-reference/processing-modules/demosaic.md)
-: Image is in raw data format with only latent colors. Each pixel carries lightness and color information for only one base color. Bear in mind that some of the modules in this part of the pipe can also act on non-raw input images in RGB format with full information on all three color channels.
+: Image is in raw data format with only latent data. Each pixel carries a single intensity for one single primary color, and camera primaries are very different from human primaries. Bear in mind that some of the modules in this part of the pipe can also act on non-raw input images in RGB format with full information on all three color channels.
```

Comment:
I'm not sure that "Image is in raw data with only latent data" really says anything. Are you ok with removing this sentence entirely?

Reply:
What I mean is that the raw file needs some reconstruction work, it's not yet an image + the raw RGB is really not RGB in the way we expect it. The formulation is perhaps unclear, but I would like to have that stated somewhere because it's important.

Reply:
How about "The raw image information does not yet constitute an "image" but merely "data" about the light captured by the camera. Each pixel carries..."

Reply:
The raw image information does not yet represent a *viewable* image, but is merely consisting of data about the light captured by the camera. Each pixel carries...

Reply:
@matt-maguire that sounds good to me